src/components/Home.vue

```diff
@@ -1,12 +1,14 @@
 <template>
   <div class="hello">
-    <h1>Welcome to the homepage!</h1>
+    <about-us/>
   </div>
 </template>
 
 <script>
+import aboutUs from './About-us'
 export default {
-  name: 'Home'
+  name: 'Home',
+  components: {aboutUs}
```

Comment:
Make your components starting with capital letter. This is the convention. e.g., AboutUs, AnimateNumber, etc.